src/tmpfiles/tmpfiles.c

```diff
@@ -2122,7 +2121,7 @@ static int create_item(Item *i) {
 
                                         creation = CREATION_FORCE;
                                 } else {
-                                        log_debug("\"%s\" is not a symlink or does not point to the correct path.", i->path);
+                                        log_warning("\"%s\" is not a symlink or does not point to the correct path.", i->path);
```

Comment:
hmm, this probably goes too far. think /etc/resolv.conf. We create a symlink there if the thing doesn't exist, but it is *totally* *OK* if people have a regular file there too. I think this needs to remain below LOG_WARN

Reply:
You're right. This would be annoying.